### ISSUE TITLE
Update "Do <input> fields' placeholders need to pass colour contrast ratio?"

### ISSUE THREAD

**Issue by Jefferydo** [cenw-wscoe/WCAG-in-Plain-Language#52]
This is a 1.4.3 Contrast issue, not [2.4.3 - Focus Order](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html)

> "If the placeholder text is all there is as instructions for the input field before typing in (such as "Write email here"), yes. However if there is a static / permanent <label> with the same instructions as text and it's linked to the input field via ID, no - it's optional. However, it is still recommended."

The thing is, you don't want to suggest that it's ok to echo the label in the placeholder text. That's redundant and creates noise in screen readers.

> "**Note: If the placeholder text is all there is as instructions before typing in AND as typing in, this is a [3.3.2 - Labels or Instructions (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html) fail."

If there's no label, only the placeholder, it's a fail. I wasn't aware that it's a fail if instructions disappear once the user starts typing. The a11y course calls the issue out with a warning instead: [Placeholder text](https://bati-itao.github.io/learning/esdc-self-paced-web-accessibility-course/module6/form-instructions.html#placeholder-text). That said, this note isn't relevant to the question about contrast. Why include it here?
